### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Kierany9**
Fixes https://github.com/HippieStationCode/HippieStation13/issues/2371 and https://github.com/HippieStationCode/HippieStation13/issues/1963

 - Nuke Ops shuttle consoles no longer break after being deconstructed.
 - Stunprods are now crafted without powercells, you'll need to find and add one of high enough capacity for it to work of course.

### CODE REVIEW

**Comment by JohnGinnane** [comment]
Something tells me the stunprod fix won't fix it. The issue stems from the requirement being `/obj/item/weapon/stock_parts/cell` as seen here: https://github.com/HippieStationCode/HippieStation13/blob/master/code/modules/crafting/recipes.dm#L48

**Comment by Kierany9** [comment]
Power cell is now added manually to tablecrafted stunprods to avoid having to add a snowflake var check into the crafting proc and because Bawb doesn't want stunprods to be craftable with power cells that are too weak.

**Comment by JohnGinnane** [comment]
Code looks good. Did you test deconstructing the shuttle consoles and also manually and table crafting stunprods?

**Comment by Kierany9** [comment]
Yes

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -44,10 +44,8 @@
 	result = /obj/item/weapon/melee/baton/cattleprod
 	reqs = list(/obj/item/weapon/restraints/handcuffs/cable = 1,
```
Comment (Carbonhell) [inline code/modules/crafting/recipes.dm]: Won't this give a prod with a cell in it

Comment (Carbonhell) [inline code/modules/crafting/recipes.dm]: I mean,the result var

File: html/changelogs/Kierany9 - Bugfixes [hunk 2]
```diff
@@ -0,0 +1,5 @@
+author: Kierany9
```
Comment (Carbonhell) [inline html/changelogs/Kierany9 - Bugfixes]: You forgot the .yml extension

File: html/changelogs/Kierany9 - Bugfixes [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
+delete-after: True
+changes: 
+- bugfix: "Nuke Ops shuttle consoles no longer break after being deconstructed."
```
Comment (Carbonhell) [inline html/changelogs/Kierany9 - Bugfixes]: And the proper indentation here too

File: code/modules/crafting/recipes.dm (hunk 1 above)
Comment (Kierany9) [inline code/modules/crafting/recipes.dm]: No, the cell was inserted in the parts list.

**Pull request merged**